### ISSUE TITLE
The script does not replace the images of the seasons of the series. 

### ISSUE THREAD

**Issue by Druidblack** [Iceshadow1404/JellyfinUpdatePoster#1]
The script does not replace the images of the seasons of the series. I'm downloading an archive from TPdb that contains the title image and the covers of the seasons.  The title image is updated seasons no. If you put the cover of the season separately, then the script applies it to the title image of the series, and not to the season.

**Comment by Iceshadow1404** [comment]
> The script does not replace the images of the seasons of the series. I'm downloading an archive from TPdb that contains the title image and the covers of the seasons. The title image is updated seasons no. If you put the cover of the season separately, then the script applies it to the title image of the series, and not to the season.

I can not recreate this Bug [Video](https://www.youtube.com/watch?v=PKX1IEyRiv8)

Feel free to message me on Discord: iceshadow_

**Comment by Druidblack** [comment]
I sent a request in the discord. Here is the link how it works for me.  If something is not visible due to the wide format, tell me, I will overwrite it.

https://youtu.be/GzCKboZpt-w

**Comment by Druidblack** [comment]
Could this be related to the language used in jf?

**Comment by Druidblack** [comment]
I did a little research. If the word season is written in another language (I tried Russian, German and French), then the cover of the season is not updated. If you correct the word season from any language to English in the metadata, it does not help (I updated the metadata after the change), apparently there is information about the language somewhere else.

P.S. However, once I managed to get such a result, during the update of the season posters, I renamed the word season from French to English and the desired cover appeared, but when the library was updated again, it disappeared. Perhaps jf deletes unnecessary folders with incorrect languages.


https://youtu.be/1rfISWmsc_A
